packages/airnode-validator/src/config/config.ts

```diff
@@ -132,15 +132,23 @@ export const disabledGatewaySchema = z
 
 export const gatewaySchema = z.discriminatedUnion('enabled', [enabledGatewaySchema, disabledGatewaySchema]);
 
-export const heartbeatSchema = z
+export const enabledHeartbeatSchema = z
```

Comment:
Do we need separate schemas for those? Or you prefer that for readability? (I personally don't care with either).